src/util/shim/socket.io.js

```diff
@@ -0,0 +1,8 @@
+export default thro
+export const Server = thro
+export const Namespace = thro
+export const Socket = thro
+
+function thro() {
+    throw new Error('transportLibrary "socket.io" is not supported in browsers. Set the `transportLibrary` option to "socket.io" only if you are use roslib in an environment like Node.js or Electron.')
+}
```

Comment:
- [ ] TODO this file is totally wrong, I didn't realize socket.io works in a browser too. Update this so that it exports the global stuff from a global `io` variable from the global-script-tag version of socket.io.